src/tag.rs

```diff
@@ -0,0 +1,723 @@
+                    "backsack 2 blue_markings 2 countershading 2 markings 2 challen 2",
+                )
+                .into(),
+                related_tags_updated_at: "2020-03-04T21:55:37.812-05:00".parse().ok(),
+                category: Category::Character,
+                is_locked: false,
+                created_at: "2020-03-05T05:49:37.994-05:00".parse().unwrap(),
+                updated_at: "2020-03-05T05:49:37.994-05:00".parse().unwrap(),
+            }),
+            Ok(Tag {
+                id: 12048,
+                name: "fish_bowl".into(),
+                post_count: 147,
+                related_tags: concat!(
+                    "aquarium 108 fish_bowl 108 vivarium 108 mammal 76 marine 71 ",
+                    "fish 69 hi_res 56 clothing 50 feral 49 male 49 female 48 ",
+                    "water 48 fur 47 text 44 hair 42 ambiguous_gender 40 anthro 40 ",
+                    "group 38 english_text 36 smile 36 simple_background 35 ",
+                    "felid 34 open_mouth 34 solo 34 clothed 32",
+                )
+                .into(),
+                related_tags_updated_at: "2021-12-11T15:59:15.118-05:00".parse().ok(),
+                category: Category::General,
+                is_locked: false,
+                created_at: "2020-03-05T05:49:37.994-05:00".parse().unwrap(),
+                updated_at: "2021-12-11T15:59:15.118-05:00".parse().unwrap(),
+            }),
+            Ok(Tag {
+                id: 12053,
+                name: "sefeiren".into(),
+                post_count: 1581,
+                related_tags: concat!(
+                    "conditional_dnp 300 sefeiren 300 female 222 male 211 claws 176 ",
+                    "anthro 173 genitals 165 mammal 165 paws 161 hindpaw 151 hi_res 150 ",
+                    "feral 149 fur 142 tongue 142 nude 131 duo 122 open_mouth 122 ",
+                    "scalie 117 bodily_fluids 116 frisky_ferals 113 horn 112 solo 112 ",
+                    "penis 108 text 104 blush 101",
+                )
+                .into(),
+                related_tags_updated_at: "2022-04-06T06:21:26.712-04:00".parse().ok(),
+                category: Category::Artist,
+                is_locked: false,
+                created_at: "2020-03-05T05:49:37.994-05:00".parse().unwrap(),
+                updated_at: "2022-04-06T06:21:26.713-04:00".parse().unwrap(),
+            }),
+            Ok(Tag {
+                id: 12052,
+                name: "skee".into(),
+                post_count: 2,
+                related_tags: concat!(
+                    "male 2 solo 2 skee 2 simple_background 2 shorts 1 bulge 1 ",
+                    "clothing 1 genital_outline 1 genitals 1 hi_res 1 huge_balls 1 ",
+                    "huge_penis 1 hyper 1 hyper_balls 1 hyper_genitalia 1 ",
+                    "hyper_penis 1 limitedvision 1 lizard 1 anthro 1 multi_genitalia 1 ",
+                    "multi_penis 1 penis 1 penis_outline 1 reptile 1 scalie 1",
```

Comment:
can we have less... explicit mock data 😅

Reply:
Oh god I'm so sorry! I didn't even read the actual content. I'll get this cleaned up.